dashboard/test/ui/features/weblab/versions.feature

```diff
@@ -0,0 +1,44 @@
+@dashboard_db_access
+@as_student
+Feature: Weblab Versions
+
+Background:
+  Given I am on "http://studio.code.org/s/allthethings/stage/32/puzzle/1?noautoplay=true"
+  Then I rotate to landscape
+  And I wait until element "#submitButton" is visible
+
+Scenario: Weblab Versions
+  # Create the initial version by adding a CSS file:
+  When I click selector "#headers div:nth-child(2)"
+  And I wait for 3 seconds
+
+  # Click save & refresh:
+  Then I click selector "#versions-header ~ div"
```

Comment:
I've never seen the [general sibling combinator selector](https://www.w3.org/TR/css3-selectors/#general-sibling-combinators) before.  TIL.